frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -2101,6 +2115,10 @@ function ReaderFooter:_updateFooterText(force_repaint, full_repaint)
         return
     end
     local text = self:genFooterText()
+    for dummy, v in pairs(self.external_footer_content) do
+        text = v() .. " " .. self:get_separator_symbol() .. " " .. text
```

Comment:
I see that your content_func() may return `""`. No risk of having concatenated separators with empty text in between?
ie. `45% | 12 / 345 | |`

Reply:
> it should be removed if empty anyway

What/how do you mean? What would remove it?